mtiapko/lab1-BasicIO/src/common.c

```diff
@@ -0,0 +1,13 @@
+#include "common.h"
+
+void delay(volatile uint32_t val)
```

Comment:
It's better to use inner cycle to avoid improper call signature:
```C
void delay(volatile uint32_t val) {
    volatile uint32_t cnt = 10000;
    while (val --> 0) {
        while (cnt --> 0);
    }
}
```